resources/index.html

```diff
@@ -9,5 +9,6 @@
     <div id="app">
     </div>
     <script src="main.js" type="text/javascript"></script>
+    <script>rest_resources_viz.core.on_jsload();</script>
```

Comment:
I could have done it that way I guess, but then figwheel would probably go crazy. Usually the tooling calls one function (`on-jsload`) at every file change.